docs/threat-actor-intel.md

```diff
@@ -0,0 +1,70 @@
+| [Home](../README.md) |
+|----------------------|
+
+# Threat Actor Intelligence
+
+The **Threat Actor Intelligence** feature in Threat Intelligence Management (TIM) provides analysts with detailed insights into adversaries, their motivations, and their tactics, techniques, and procedures (TTPs).
+
+## What is Threat Actor Intelligence
+
+A *threat actor* is an individual, group, or organization responsible for conducting malicious cyber activities. FortiGuard provides continuously updated profiles of threat actors that include their capabilities, targeted sectors, geographical scope, and known campaigns.
+
+By integrating this intelligence into TIM, SOC analysts gain enriched context that goes beyond indicators or feeds—linking observed activity to known adversaries and their historical behaviors.
+
+## How Threat Actor Data is Used in TIM
+
+- **Correlation with Threat Reports**: TIM automatically correlates FortiGuard threat actor data with existing feeds, reports, and ingested observables. Analysts can view connections between threat actors and ongoing campaigns, giving greater context to alerts.
+
```

Comment:
Add one more point for "Mapping to CVEs" , The threat actor record also gets correlates with CVEs associated with  the threat actors